### PR TITLE
.github/labeler: add build/ops label

### PR DESCRIPTION
**Pull request by batrick**
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by batrick** [comment]
jenkins test make check

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,3 +1,18 @@
+build/ops:
+  - **/CMakeLists.txt
```
Comment (tchaikov) [inline .github/labeler.yml:2]: could you add `admin/**` here?

**Review by tchaikov — APPROVED** [review]

File: .github/labeler.yml [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+  - cmake/**
+  - debian/**
```
[the remark this answers is not in the document]
Comment (batrick) [reply .github/labeler.yml:4]: done, thanks

File: .github/labeler.yml (hunk 1 above)
Comment (batrick) [reply .github/labeler.yml:2]: done, thanks

**Pull request merged**